webpack.config.js

```diff
@@ -27,15 +32,19 @@ module.exports = {
       {
         test: /\.scss$/,
         use: [
-          ExtractCssChunks.loader,
+          // Instead of style-loader, to avoid FOUC
+          MiniCssExtractPlugin.loader,
```

Comment:
This is the far more common CSS extractor plugin, and I was able to get it to behave this time.